config/resources.txt

```diff
@@ -40,8 +40,15 @@ ASSET_CDN_URL http://localhost:7123/
 ########################
 # Asset Caching Config #
 ########################
-## Assets can opt-in to caching their results into `tmp`.
-## This is important, as preferences assets take upwards of 30 seconds (without sleeps) to collect.
-## The cache is assumed to be cleared by TGS recompiling, which deletes `tmp`.
-## This should be uncommented on production
-#CACHE_ASSETS
+## Assets can opt-in to caching their results into `data/spritesheets/legacy_cache`.
+## This is useful, as preferences assets take upwards of 30 seconds (without sleeps) to collect.
+## The cache can be manually cleared via the "Regenerate Asset Cache" verb.
+## This is on in development now, because it is only used by a few select spritesheets.
+CACHE_ASSETS
```

Comment:
I believe we wanted this disabled by default to prevent caching during development

```suggestion
#CACHE_ASSETS
```

Reply:
The entire purpose of this PR is to allow caching on development - it reduces init times.

There are only *two* spritesheets using the legacy cache. Species and antagonist preview for prefs. They are not changed often and only affect a few coders, while init times affect everyone. Hence I declared it 'worth it'. I've also added a verb to immediately regenerate the cached spritesheets for developers. This is also why I moved the cache out of `tmp` and into `data`, so it persists. Production will also need to run this verb, but it's rare enough that I'm not really concerned about it. The worst that happens is some icon in prefs will show as another icon until somebody presses the verb.

As for the smart cache, it is designed to be enabled on development. It is faster than generating the spritesheet, even with rust, and compares all the source hashes, so it can *always* validate if the cache is correct.